downloads/swan-lake-release-notes/swan-lake-2201.7.0/RELEASE_NOTE.md

```diff
@@ -82,6 +100,34 @@ public function main() {
 
 ### Improvements
 
+#### Support for constant declarations without a type descriptor
+
+Constants can now be declared without specifying a type descriptor. Previously it was required to specify a type descriptor except with numeric and string literal expressions.
+
+Both of the following cases are allowed now.
+
+```ballerina
+const map<string> HTTP_OK = {httpCode: "200", message: "OK"};
+const HTTP_BAD_REQUEST = {httpCode: "400", message: "BAD REQUEST"};
+```
+
+#### Support for unions as type descriptors in constant declarations
+  
+Constant declarations with unions as type descriptors are now supported.
```

Comment:
```suggestion
- Constant declarations with unions as type descriptors are now supported.
```

Reply:
Let's remove the bullet points based on https://github.com/ballerina-platform/ballerina-dev-website/pull/7157#discussion_r1232013819.